doc/nrf/libraries/modem/lte_lc.rst

```diff
@@ -64,7 +64,7 @@ The following block of code shows how the link controller API can be used to est
    	}
    }
 
-   void main(void)
```

Comment:
Missing return statement in the end of the function and return statements within the error branches are without the return value.

All examples in this file should be checked.

Reply:
> Missing return statement in the end of the function

That's on purpose. The `/* Continue execution ... */` comment covers this.

> and return statements within the error branches are without the return value.

Good catch, will fix, thanks.

Reply:
> That's on purpose. The `/* Continue execution ... */` comment covers this.

Well, it could but the code example looks like it's been properly compiling without modifications and now it will at least give a compiler warning (if not error) due to missing return statement.